### PR TITLE
de-duplicate moduels

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/core/azure-core/azure/core/__init__.py [hunk 1]
```diff
@@ -24,14 +24,12 @@
...
+from ._version import VERSION
 __version__ = VERSION
 
-from .configuration import Configuration  # pylint: disable=unused-import
-from .pipeline_client import PipelineClient
-from .pipeline.transport import HttpRequest  # pylint: disable=unused-import
-from .exceptions import HttpResponseError  # pylint: disable=unused-import
+from ._pipeline_client import PipelineClient
 from ._match_conditions import MatchConditions
+from .exceptions import HttpResponseError # pylint: disable=unused-import
```
Comment (lmazuel) [inline sdk/core/azure-core/azure/core/__init__.py]: Let's discuss this one with @johanste :)

File: eng/CHECKENFORCER [hunk 2]
```diff
@@ -1,2 +1,2 @@
 format: v0.1-alpha
-minimumCheckRuns: 1
```
Comment (weshaggard) [inline eng/CHECKENFORCER]: This looks like a bad merge, please undo. fyi @mitchdenny

Comment (xiangyan99) [reply eng/CHECKENFORCER]: Done.

**Pull request merged**